modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/sts/TestPassiveSTS.java

```diff
@@ -313,10 +362,17 @@ private void setSystemProperties() {
         return claimMappingList.toArray(new ClaimMapping[0]);
     }
 
-    private boolean requestMissingClaims (HttpResponse response) {
+    private boolean requestMissingClaims(HttpResponse response) {
 
         String redirectUrl = Utils.getRedirectUrl(response);
         return redirectUrl.contains("consent.do");
     }
 
+    private String getPassiveParamsForTenant(String params) {
```

Comment:
Better to rename the method to something like appendTenantDomainQueryParam or something along the line.

Reply:
addressed